lib/dependencyManagement.ts

```diff
@@ -150,9 +150,18 @@ async function getProjectPackageJsonLocations() {
   return packageParentDirs;
 }
 
+export async function packagesNeedInstalled(directory) {
```

Comment:
Wonder if we could come up with a slightly more informative name for this one. Some suggestions are `areDependenciesUpToDate()`, `detectAvailableDependencyUpdates()`, `areNewDependenciesAvailable()`

Reply:
I agree we could have a better name, but this isn't checking for updates, it is checking the any packages would be installed.  `update` implies that there may new versions available and that is not a part of the check.  Maybe something like `hasMissingPackages`